packages/angular/projects/vcd/plugin-builders/src/lib/base/index.ts

```diff
@@ -60,13 +69,17 @@ async function commandBuilder(
     const pluginLibsBundles = new Map<string, string>();
 
     // Get the configuration
-    const config = configs.config[0];
+    const config = configs.config[0] || configs.config as webpack.Configuration;
 
     // Make sure we are producing a single bundle
-    delete config.entry.polyfills;
+    delete config.entry['polyfills'];
```

Comment:
Here and all other similar changes:

Isn't it possible to preserve the '.' property access?
It is safer than the '[prop_name]' one

Reply:
The problem is that in webpack `entry?: string | string[] | Entry | EntryFunc;` where
`interface Entry {
        [name: string]: string | string[];
    }`
and therefore dot syntax doesn't work.